apps/datagouvfr/lib/datagouvfr/client/api.ex

```diff
@@ -5,8 +5,13 @@ defmodule Datagouvfr.Client.API do
   require Logger
   use Datagouvfr.Client
 
+  alias Datagouvfr.Client.Request.Page
+
   @type response :: {:ok, any} | {:error, any}
 
+  # HTTP client injection. Allow mock injection in tests
+  defp http_client, do: Application.get_env(:transport, :httpoison_impl)
```

Comment:
Bien joué :-)

Reply:
Attention à utiliser plutôt `fetch_env!`, car `get_env` peut retourner `null`:

```
iex(1)> Application.get_env(:transport, :foobar)         
nil
iex(2)> Application.fetch_env!(:transport, :foobar)
** (ArgumentError) could not fetch application environment :foobar for application :transport because configuration at :foobar was not set
    (elixir 1.12.2) lib/application.ex:669: Application.fetch_env!/2
```

Reply:
tu as vu cette remarque @andlio ?